app/managers/network_manager.ts

```diff
@@ -129,12 +153,23 @@ class NetworkManager {
             logDebug('Invalid SSL certificate:', event.errorDescription);
             const parsed = urlParse(event.serverUrl);
             Alert.alert(
-                getLocalizedMessage(DEFAULT_LOCALE, t('server.invalid.certificate.title'), 'Invalid SSL certificate'),
-                getLocalizedMessage(
-                    DEFAULT_LOCALE,
-                    t('server.invalid.certificate.description'),
-                    'The certificate for this server is invalid.\nYou might be connecting to a server that is pretending to be “{hostname}” which could put your confidential information at risk.',
-                ).replace('{hostname}', parsed.hostname),
+                this.intl.formatMessage(messages.invalidSslTitle),
+                this.intl.formatMessage(messages.invalidSSLDescription, {hostname: parsed.hostname}),
+            );
+        } else if (SERVER_TRUST_EVALUATION_FAILED === event.errorCode && !showingServerTrustAlert) {
+            logDebug('Invalid SSL Pinning:', event.errorDescription);
+            showingServerTrustAlert = true;
+            Alert.alert(
+                this.intl.formatMessage(messages.invlidePinningTitle),
```

Comment:
```suggestion
                this.intl.formatMessage(messages.invalidePinningTitle),
```

Reply:
FWIW The above recommendation also has a typo. "invalide"